python/mxnet/optimizer.py

```diff
@@ -194,6 +209,15 @@ def update(self, index, weight, grad, state):
         """
         raise NotImplementedError()
 
+    @learning_rate.setter
+    def learning_rate(self, lr):
```

Comment:
Let's use set_learning_rate explicitly rather than a property setter.

Reply:
and add documentation to the function

Reply:
resolved